lib/adapters/parsers/csv.js

```diff
@@ -33,7 +33,7 @@ class CSVParser extends base {
                     self.doneParsing = true;
                     emit(buffer);
                     buffer = [];
-                    csvStream.end();
+                    csvStream.emit('end');
```

Comment:
Yup this was completely accidental change and great catch, now removed from this PR.